dlhub_sdk/client.py

```diff
@@ -110,11 +111,10 @@ def __init__(self, dlh_authorizer: Optional[GlobusAuthorizer] = None,
             search_authorizer = auth_res['search']
 
         # Define the subclients needed by the service
-        self._fx_client = FuncXClient(fx_authorizer=fx_authorizer,
-                                      search_authorizer=search_authorizer,
-                                      openid_authorizer=openid_authorizer,
-                                      no_local_server=kwargs.get("no_local_server", True),
-                                      no_browser=kwargs.get("no_browser", True))
+
+        login_manager = FuncXLoginManager(authorizers=auth_res)
```

Comment:
tests are failing because `auth_res` is undefined if the user provides a `fx_authorizer` to the client. Can `auth_res` here just be replaced by `fx_authorizer`, or does more unpacking need to happen?